lang/fortran/README.md

```diff
@@ -0,0 +1,4 @@
+Interface only for Ubuntu, otherwise you need to get the lgfortran.a file. To run the app first you need to install the following packages: 
```

Comment:
```suggestion
Interface only for Ubuntu, otherwise you need to get the lgfortran.a file.
To run the app first you need to install the following packages: 
```